### PR TITLE
Add doc about env profile

### PR DESCRIPTION
**Pull request by mszostok**
## Description

Changes proposed in this pull request:

- Add doc about env profile

## Testing

https://add-profile-env.botkube-docs-58l.pages.dev/next/env-profile

## Related issue(s)

- https://github.com/kubeshop/fuse/issues/154

### CODE REVIEW

**Comment by mszostok** [comment]
Note: I decided to use tabs in both places because displaying those GIFs one under the other caused too much blinking and was unpleasant to read.

File: docs/assets/profile-usecase-helm.gif
Comment (pkosiec) [inline docs/assets/profile-usecase-helm.gif:1]: 14.5MB is quite huge - can we reduce the size?

I used [gifsicle](https://github.com/kohler/gifsicle), with the following command:

```
gifsicle -i prompt.gif -O4 --colors 256 --resize-width 958 -o prompt3.gif
```

Of course you can tune the params but basically 958px is max width for our docs website  👍

File: docs/assets/profile-usecase-scale.gif
Comment (pkosiec) [inline docs/assets/profile-usecase-scale.gif:1]: same here

File: docs/env-profile.mdx [hunk 1]
```diff
@@ -0,0 +1,87 @@
+---
+id: env-profile
+title: Environment profile
+sidebar_position: 8
+---
+
+# Environment profile
+
+To mitigate AI unpredictability, we have implemented several strategies to enhance its behavior and reliability. This page outlines the environment-specific details attached to chat sessions.
```
Comment (pkosiec) [inline docs/env-profile.mdx:9]: Can we rephrase it a bit? IMO unpredictability isn't relevant here, it's more about response quality? 🤔 
 
```suggestion
To enhance AI response quality and reliability, we have implemented several strategies. This page outlines one of them—the environment-specific details attached to Fuse chat sessions.
```

**Review by pkosiec — APPROVED** [review]
👌

**Pull request merged**